sceptre/scipool/templates/sc-tag-options.j2

```diff
@@ -30,6 +30,12 @@ Resources:
       Key: "CostCenter"
       Value: {{ CostCenter }}
   {% endfor %}
+  OtherCostCenterTag:
+    Type: AWS::ServiceCatalog::TagOption
+    Properties:
+      Active: true
+      Key: "CostCenter"
+      Value: "Other / 000001"
```

Comment:
this seems like a bad idea to me.  i think all id/codes should come from a central source of truth.  this effectively breaks that pattern.  What if we change the ID name or number?

Reply:
the "other" value is only used by service catalog as a place holder. now that i've split the two lambdas apart i was thinking about what needs the "other" value: we don't need it for the cost rules lambda because we don't want an "other" cost category, and we don't want it in the list we show to users either. but i see your point about the source of truth. i guess i'll need to modify the lambda to treat "other" and "no program" differently since we do want "no program" in the other cases too.